app/services/workers/send-technical-help.js

```diff
@@ -12,32 +13,40 @@ module.exports.execute = function (task) {
   }
 
   if (Config.ZENDESK_ENABLED === 'true') {
-    const zendesk = new Zendesk({
-      url: Config.ZENDESK_API_URL,
-      email: Config.ZENDESK_EMAIL_ADDRESS,
-      token: Config.ZENDESK_API_KEY
-    })
-
     if (Config.ZENDESK_TEST_ENVIRONMENT === 'true') {
       subjectText = 'Test: Help With Prison Visits - Help'
       tagText = ['HelpWithPrisonVisits', 'Test']
     }
 
-    return zendesk.tickets.create({
-      submitter_id: '114198238551',
-      requester: {
-        name: personalisation.name,
-        email: personalisation.contactEmailAddress,
-        verified: true
-      },
-      subject: subjectText,
-      comment: {
-        body: personalisation.issue
-      },
-      tags: tagText
-    }).then(function (result) {
-      console.log('Zendesk ticket, ' + result.ticket.id + ' has been raised')
-    })
+    const zendeskApiUrl = Config.ZENDESK_API_URL + '/api/v2/tickets.json'
+
+    const headers = {
+      Authorization: `Basic ${Buffer.from(Config.ZENDESK_EMAIL_ADDRESS +
+        '/token:' + Config.ZENDESK_API_KEY).toString('base64')}`
```

Comment:
Only comment for tidiness sake is maybe move `Config.ZENDESK_EMAIL_ADDRESS +
        '/token:' + Config.ZENDESK_API_KEY` into it's own var to make it easier to read?